src/@next/components/molecules/ProductTile/test.tsx

```diff
@@ -0,0 +1,14 @@
+import { shallow } from "enzyme";
+import "jest-styled-components";
+import React from "react";
+
+import { ProductTile } from ".";
+import { PRODUCT } from "./fixtures";
+
+describe("<ProductTile />", () => {
+  it("exists", () => {
+    const wrapper = shallow(<ProductTile product={PRODUCT} />);
+
```

Comment:
Lets add few more tests:

- does it have price displayed
- does it have name of product displayed for ex.